src/components/UserMenu/UserMenuSignedIn.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useCallback } from 'react'
```

Comment:
There's little contrast between the opened menu and the background. It's hard to understand where the menu ends. What if we add a glow or something to make it more evident? If not, this might be an accessibility issue as well:
<img width="596" alt="Screenshot 2023-12-29 at 12 48 52" src="https://github.com/decentraland/ui/assets/1120791/f1bb0704-ec16-469c-b30d-bc2d7aa3062c">